docs/sources/mimir/references/http-api/index.md

```diff
@@ -238,6 +239,14 @@ GET <alertmanager-http-prefix>/api/v1/status/buildinfo
 
 This endpoint returns in JSON format information about the build and enabled features. The format returned is not identical, but is similar to the [Prometheus Build Information endpoint](https://prometheus.io/docs/prometheus/latest/querying/api/#build-information).
 
+### Format query
+
+```
+GET,POST <prometheus-http-prefix>/api/v1/format_query?query=up
+```
+
+Formats the PromQL query, see [Prometheus documentation](https://prometheus.io/docs/prometheus/latest/querying/api/#formatting-query-expressions).
```

Comment:
Let's follow the same pattern as for the other API endpoints:

```suggestion
Formats the PromQL query.

This endpoint is compatible with the Prometheus format query endpoint.

For more information about formatting queries, refer to Prometheus [formatting query expreessions](https://prometheus.io/docs/prometheus/latest/querying/api/#formatting-query-expressions).
```

Reply:
Also, does this endpoint require authentication like the others do? If so, we should mention that here.